ApplicantQualityScore.md

```diff
@@ -0,0 +1,104 @@
+Applicant Quality Score Service
+====================
+
+Table of Contents
+_____________
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+Summary
+-----------
+Applicant Quality Score Service is a service helps JobSeekers check how well their profiles match job requirements before applying.
+
+For each job you add, we provide a relevance score:
+
+* 1 - Not Relevant
+* 2 - Low Relevant
+* 3 - Relevant
+* 4 - High Relevant
+
+The Applicant Quality Score Service is available at `/core/application-quality-score`.
+
+Request Structure
+-----------
+
+Requests consist of:
+
+| Parameter             | Require/Optional | Type            | Description                               |
+|:----------------------|:-----------------|:----------------|:------------------------------------------|
+| `email `              | Yes              | String          | JobSeeker Email                           |
+| `job_did `            | yes              | Array of String | An array of job DIDs for the application. |
+
+
+Example cURL request : 
+
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/application-quality-score \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "email":"email@example.com",
+        "job_did": ["J8B2606V22T551FGNWF","J8B36Z60T1523M7FNNN","J7X1RQ68Q8Z3QX327WK","J8D8FN7111Q70X662Z1","JHN0YB6BKFL70JB7LB0","JD67L36K6XGYLD9LBGD","JD825Z6WS1GH0QP7KDC"]
+      }'
+```
+
+Response Structure
+-----------
+Upon a successful execution, the response will have a status code of 200 and a JSON content with the following structure:
+
+job_did (string): The job ID for which the quality score is calculated.
+relevant (number): Indicates the relevance of the applicant to the job.
+
```

Comment:
What is the score then? And what is the relationship between score and relevancy? We don't seem to be sorting the results by the score, so what is the score? Is it just another representation of the `relevant` field?

Reply:
@glpayson  that's correct, this has been added as an extra field in order to sort them in the future if needed.